<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="Admin" name="testNGpractice.testNGpractice.GroupingTestcase" tests="9" failures="1" timestamp="2020-11-16T12:36:15 IST" time="0.029" errors="0">
  <testcase name="BillDueexpire" time="0.016" classname="testNGpractice.testNGpractice.GroupingTestcase"/>
  <system-out/>
  <testcase name="AddVendor" time="0.005" classname="testNGpractice.testNGpractice.GroupingTestcase">
    <failure type="org.junit.ComparisonFailure" message="expected:&lt;[G]mail&gt; but was:&lt;[]mail&gt;">
      <![CDATA[org.junit.ComparisonFailure: expected:<[G]mail> but was:<[]mail>
at org.junit.Assert.assertEquals(Assert.java:115)
at org.junit.Assert.assertEquals(Assert.java:144)
at testNGpractice.testNGpractice.GroupingTestcase.AddVendor(GroupingTestcase.java:23)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(Unknown Source)
at org.testng.TestRunner.privateRun(TestRunner.java:764)
at org.testng.TestRunner.run(TestRunner.java:585)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1218)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
at org.testng.TestNG.runSuites(TestNG.java:1069)
at org.testng.TestNG.run(TestNG.java:1037)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- AddVendor -->
  <system-out/>
  <testcase name="AddProduct" time="0.002" classname="testNGpractice.testNGpractice.GroupingTestcase"/>
  <system-out/>
  <testcase name="SearchProduct" time="0.003" classname="testNGpractice.testNGpractice.GroupingTestcase"/>
  <system-out/>
  <testcase name="FundTransfer" time="0.003" classname="testNGpractice.testNGpractice.GroupingTestcase"/>
  <system-out/>
  <testcase name="BillPaid" time="0.000" classname="testNGpractice.testNGpractice.GroupingTestcase"/>
  <system-out/>
  <testcase name="DeleteSearch" time="0.000" classname="testNGpractice.testNGpractice.GroupingTestcase"/>
  <system-out/>
  <testcase name="DeleteProduct" time="0.000" classname="testNGpractice.testNGpractice.GroupingTestcase"/>
  <system-out/>
  <testcase name="DeleteVendor" time="0.000" classname="testNGpractice.testNGpractice.GroupingTestcase"/>
  <system-out/>
</testsuite> <!-- testNGpractice.testNGpractice.GroupingTestcase -->
